src/components/table/partial-styles/tabulator-custom-styles.scss

```diff
@@ -67,8 +67,7 @@
     }
 
     .tabulator-cell {
-        height: functions.pxToRem(44);
-        line-height: functions.pxToRem(32);
+        height: functions.pxToRem(44) !important;
```

Comment:
is it possible to avoid using `!important` in this context?

Reply:
I have tried to avoid using `!important` as much as it was possible. But because we're overriding Tabulator's original styles, sometimes it is not possible. Tabulator has a messy and bad-written CSS unfortunately. So it's hard to avoid these things.

This one is also a very specific case, so using !important is not a very big deal. To avoid it, we have to add a lot of more classes before `.tabulator-cell` to make the styles very specific. But that's not worth it. Because our own CSS becomes harder to read & maintain, and the output becomes larger. It's better to keep the CSS structure clean and do not nest classes inside each other.